src/app/containers/ConsentBanner/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Consent Banner Container should correctly render amp banner 1`] = `null`;
```

Comment:
Not sure if I prefer a null snapshot or not having the test for the amp version 🤔

Reply:
Actually, this prove the platform context config is correct so lets keep it